mediapicker/src/main/res/layout/oss_fragment_main.xml

```diff
@@ -1,37 +1,61 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
convert it to constraint layout

Reply:
i already make constraint but viewpager creates some kind of problem 

<?xml version="1.0" encoding="utf-8"?>
<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
    xmlns:app="http://schemas.android.com/apk/res-auto"
    android:id="@+id/constraint_layout_parent"
    android:layout_width="match_parent"
    android:layout_height="match_parent">

    <com.google.android.material.tabs.TabLayout
        android:id="@+id/tabLayout"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:layout_constraintTop_toTopOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        app:tabBackground="@android:color/white"
        app:tabIndicatorColor="@color/primary"
        app:tabSelectedTextColor="@color/primary"
        app:tabTextColor="@color/oss_tab_text_color" />

    <LinearLayout
        android:id="@+id/permission_access_management"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        android:gravity="end"
        app:layout_constraintTop_toBottomOf="@id/tabLayout"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintEnd_toEndOf="parent">

        <androidx.appcompat.widget.AppCompatButton
            android:id="@+id/access_management"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:layout_marginEnd="8dp"
            android:padding="4dp"
            android:text="@string/request_partial_access" />

        <androidx.appcompat.widget.AppCompatButton
            android:id="@+id/full_access_button"
            android:layout_width="wrap_content"
            android:layout_height="wrap_content"
            android:padding="4dp"
            android:text="@string/request_full_access" />
    </LinearLayout>

    <androidx.viewpager.widget.ViewPager
        android:id="@+id/viewPager"
        android:layout_width="match_parent"
        android:layout_height="0dp"
        app:layout_constraintTop_toBottomOf="@id/permission_access_management"
        app:layout_constraintBottom_toTopOf="@id/action_button"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        android:layout_weight="1" />

    <androidx.appcompat.widget.AppCompatButton
        android:id="@+id/action_button"
        style="@style/OSS_CustomButton"
        android:layout_width="match_parent"
        android:layout_height="wrap_content"
        app:layout_constraintBottom_toBottomOf="parent"
        app:layout_constraintStart_toStartOf="parent"
        app:layout_constraintEnd_toEndOf="parent"
        android:animateLayoutChanges="true"
        android:enabled="true"
        android:text="@string/oss_posting_next" />

</androidx.constraintlayout.widget.ConstraintLayout>